lib/AdminPages/Pages/Newsletters.php

```diff
@@ -100,9 +100,7 @@ public function __construct(
     $this->segmentsListRepository = $segmentsListRepository;
   }
 
-  public function render() {
-    global $wp_roles; // phpcs:ignore Squiz.NamingConventions.ValidVariableName.NotCamelCaps
-
+  public function redirect(): void {
```

Comment:
I find the name a bit confusing since it doesn't redirect every time. So maybe we could rename it to something like `checkStatsRedirect` and return URL to redirect or `maybeStatsRedirect` or maybe even move the whole code to Menu.php since it works kind of as a router.

Reply:
Good point. I will change it. I thought about routing as a part of Menu.php, but I liked that the redirect logic stays in the same place. So, I would move the whole routing to the Menu.